lisa/utils.py

```diff
@@ -592,6 +649,21 @@ def get_help(self, style=None):
         )
 
 class LevelKeyDesc(KeyDescBase, Mapping):
+    """
+    Key descriptor defining a hierarchical level in the configuration.
+
+    Children keys will get this key assigned as a parent when passed to the
```

Comment:
Nit: the style we've gone for so far is 
[One-liner]
[Param list]
[More words]

This makes the param list always show up high in the docstring, which is helpful when it shows up in your IDE, and if you really want more info you're free to scroll down.